sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -8,6 +8,7 @@ export interface CreateApplicationDto {
    * with this application.
    */
   associatedFiles: string[];
+  applicationNumber: string
```

Comment:
This DTO is being shared for API and service classes, but the applicationNumber doesn`t belong to the API layer.
I would recommend creating a new DTO for the service or passing the info to the service with parameters.